backend/src/main/java/club/devcord/devmarkt/services/QuestionService.java

```diff
@@ -114,11 +112,11 @@ public boolean deleteQuestion(String templateName, int number) {
   Eg. 1, 2, 4 -> 1, 2, 3 (offset 0)
       1, 2, 4 -> 1, 2, 5 (offset 2)
    */
-  public void reorderQuestions(int templateId, int from, int offset) {
+  public void reorderQuestions(int templateId, int from, int offset, boolean order) {
```

Comment:
The method is named `re**order**Questions` and has a `order` parameter? Feels weird

Reply:
The order parameter controls whether the questions should be stored backwards or forwards. Since we do not delete them before, there would be conflicts with the unique index here.